README.md

```diff
@@ -1,155 +1,116 @@
-Scvim
-=====
+SCVim (> 3.5)
+==============
 
-Created by Alex Norman, it is in its early stages, use at your own risk
-This is SCVIM-git
-The webpage is here: http://www.x37v.info/projects/scvim
-If you have improvements/suggestions email `alex at x37v dot info`
+A vim plugin for supercollider.
 
+This is based of the original [scvim by Alex Norman](http://www.x37v.info/scvim/).
 
-Requirements
-------------
-
-- vim
-- supercollider (sclang)
-- ruby (sclangpipe_app, scvim)
-- cmake
-- pidof (for killing spawned processes, when closing)
-
-### Optional:
-
-- gvim or MacVim
-
-
-Installation steps
-------------------
+Features
 
-Note: If you installed SuperCollider from source and didn't disable scvim when running cmake, scvim should already be installed.
+* Syntax Highlighting
+* A faked REPL via the terminal
+* Code Navigation (jump to definitions) via a global tags file "~/.sctags"
+* Auto-Completion of known words via tag completion
+* Launch the SuperCollider QT Help System
 
-If you'd like to install scvim directly, without using the whole SuperCollider install process you can do:
 
-```
-$> cmake . && sudo make install
-```
-
-You will have to enter your password in order to finalize the installation.
-
-Some people prefer to use a build directory [though scvim doesn't actually build anything, but cmake does create some output files] to do that, from the scvim directory, type:
-
-```
-$> mkdir build && cd build
-$> cmake .. && sudo make install
-```
-
-Now you should be ready to use scvim, e.g. by running `scvim` or `scvim -g` from a command prompt.
+Installation:
+------------
 
+### `scvim` plugin installation
 
-Optional extra installation steps
----------------------------------
+It is highly recommended to use a plugin manager to install `scvim`. Most common ones are:
 
-* If you want to be able to read supercollider files with syntax highlighting, 
-have the ability to start the language and have the keybindings you've set
-up during a normal vim editing session [without launching as 'scvim'] then
-source your scvimrc file in your `~/.vimrc`.
+* [pathogen](https://github.com/tpope/vim-pathogen)
+* [vundle](https://github.com/VundleVim/Vundle.vim)
+* [vim-plug](https://github.com/junegunn/vim-plug)
 
-For example, if you have a `~/.scvimrc` file you could put this line in your `~/.vimrc`:
+### `SCVim.sc`
 
-   `so ~/.scvimrc`
+If your SuperCollider version wasn't build with vim support (i.e. if the class
+`SCVim` does not exist), you'll need to symlink `sc/SCVim.sc` somewhere where
+SuperCollider can find it. Typically that would be in `Platform.userExtensionDir`
+(from within SuperCollider).
```

Comment:
We could also note that you could add `~/.vim/bundle/scvim/sc` to `includePaths:` in `sclang_conf.yaml`. Might be easier than making a symlink maybe?